### DISCUSSION TITLE
Integration with test coverage tools/plugins

### DISCUSSION THREAD

**Discussion by cardosojc** [JetBrains/Qodana#64] [Ideas]
Are there any plans for this ? For example, with jacoco, etc..

**Comment by brichbash** [comment]
We are planning to support such integrations. We have an issue [on our tracker](https://youtrack.jetbrains.com/issue/QD-2122) for this feature. Please vote for it to get notified when there are any updates.

**Comment by cardosojc** [comment]
Nice! Is there any rough estimation when this would be available ?

brichbash ***@***.***> escreveu no dia quarta, 26/01/2022
à(s) 12:29:

> We are planning to support such integrations. We have an issue on our
> tracker <https://youtrack.jetbrains.com/issue/QD-2122> for this feature.
> Please vote for it to get notified when there are any updates.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JetBrains/Qodana/discussions/64#discussioncomment-2050504>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB5WTEHIFJFRPZKVYDOYXNLUX7SL3ANCNFSM5M22WRHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
José Cardoso

**Reply by tiulpin** [reply]
Currently, no estimations
